pkg/assets/asset.go

```diff
@@ -262,3 +262,23 @@ func XMAInfo() models.AssetInfo {
 
 	return assetInfo
 }
+
+// RemoveJSONFile removes the assets.json file if it exists.
+func RemoveJSONFile() error {
```

Comment:
if it's only used in the tests, should we put this function in a test file? what is the use case of deleting the json assets file aside testing?